sss/process_foms.c

```diff
@@ -408,6 +408,8 @@ static int ss_bytecount_stats_ingest(struct m0_cob_domain *cdom,
 	if (rc == 0) {
 		m0_buf_init(&rep->sspr_bckey, key_buf->b_addr, key_buf->b_nob);
 		m0_buf_init(&rep->sspr_bcrec, rec_buf->b_addr, rec_buf->b_nob);
+		m0_free(key_buf);
+		m0_free(rec_buf);
 	}
 
```

Comment:
This function expects the caller to free it, but its very likely that the caller would be unaware of allocation happening inside the function. I think we need to introduce something like init and fini functions for these pointer allocations for better code understanding.

Also let add comments for this function.

Reply:
Changed the approach now, m0_cob_bc_entries_dump() does not allocate buffer pointers now.